### PR TITLE
UPSTREAM: 0000: remove camelCase from resource shortcuts

### PR DESCRIPTION
**Pull request by juanvallejo**
Some openshift resources written in camelcase prevented a resource from being mapped to its alias when doing `oc get ...`.

Addresses BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1486210

cc @openshift/cli-review

### CODE REVIEW

**Comment by juanvallejo** [comment]
@fabianofranz friendly ping - wondering if this patch is something we want in origin (solves https://bugzilla.redhat.com/show_bug.cgi?id=1486210)

**Comment by fabianofranz** [comment]
@deads2k @liggitt PTAL

File: vendor/k8s.io/kubernetes/pkg/kubectl/kubectl.go [hunk 1]
```diff
@@ -140,31 +140,31 @@ var ResourcesShortcutStatic = []ResourceShortcuts{
 	},
 	{
 		ShortForm: schema.GroupResource{Group: "extensions", Resource: "psp"},
-		LongForm:  schema.GroupResource{Group: "extensions", Resource: "podSecurityPolicies"},
+		LongForm:  schema.GroupResource{Group: "extensions", Resource: "podsecuritypolicies"},
 	},
 	{
 		ShortForm: schema.GroupResource{Group: "apps.openshift.io", Resource: "dc"},
-		LongForm:  schema.GroupResource{Group: "apps.openshift.io", Resource: "deploymentConfigs"},
+		LongForm:  schema.GroupResource{Group: "apps.openshift.io", Resource: "deploymentconfigs"},
```
Comment (deads2k) [inline vendor/k8s.io/kubernetes/pkg/kubectl/kubectl.go]: didn't I eliminate this entire block in 3.6?

**Comment by deads2k** [comment]
I'd like to remove all hardcoded resource types like that.  Making them lowercase is fine though.

**Pull request closed without merging**